src/components/DefineCompany/BusinessContactInfo.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div>
+    <div v-if="!isEditing">
+      <div>Email Address: {{contact.email}}</div>
```

Comment:
Do you alternate values in case we are in summary mode and some property hasn't been entered?

For example, `{{ contact.email || '-' }}` ? (Don't implement this. I'm just asking if something is needed.)